libnavigation-base/src/main/java/com/mapbox/navigation/base/route/model/Route.kt

```diff
@@ -7,7 +7,7 @@ data class Route(
     val geometry: String?,
     val weight: Double?,
     val weightName: String?,
-    val legs: RouteLegsNavigation?,
+    val legs: List<RouteLegNavigation>?,
```

Comment:
Why did we remove `RouteLegsNavigation` wrapper?

Reply:
I'd like to have this mirroring like in `DirectionRoute`. I believe it's more clean way